util/info.c

```diff
@@ -228,29 +229,55 @@ static const char *param_type(enum fi_param_type type)
 	}
 }
 
+static char * get_var_prefix(const char *prov_name)
+{
+	int i;
+	char *prefix;
+
+	if (!prov_name)
+		return NULL;
+	else {
```

Comment:
nit: Use braces on both sides of the else if either side needs it.  I.e. add { } to the if portion.  Although in this case, just remove the else claus and out-dent the code.